### PR TITLE
Add GOVERNANCE and related files for ASWF project compliance

### PR DESCRIPTION
**Pull request by garyo**
These were originally copied from OpenEXR and modified to suit our project and process.

### CODE REVIEW

**Comment by garyo** [comment]
NOTE: We also need to publish the charter and related docs in an `ASWF` subdir, or elsewhere.

**Comment by revisionfx** [comment]
I thought we were using OSL not EXR...

**Comment by garyo** [comment]
> I thought we were using OSL not EXR...

OK, I will fix

**Comment by garyo** [comment]
> > I thought we were using OSL not EXR...
> 
> OK, I will fix

Done; see ac05544 -- let me know what you think.

**Review by revisionfx — COMMENTED** [review]
Implementation (by one plug-in and one host)  
:  maybe not coming from same source code tree?

**Review by revisionfx — APPROVED** [review]
Sorry for repeats, did not realize all files were under same review

I tried to approve code of conduct

**Review by revisionfx — DISMISSED** [review]
Implementation (by one plug-in and one host)  
:  maybe not coming from same source code tree?

**Comment by garyo** [comment]
I agree re: GOVERNING remove Chief Architect role. Done in e6f5b03. 
Pierre, you can directly comment on lines in the files -- that makes it easier to find what you're talking about.

**Comment by garyo** [comment]
> Implementation (by one plug-in and one host) : maybe not coming from same source code tree?

Fixed in 48c9ac4 -- see https://github.com/ofxa/openfx/blob/feature/aswf-files/STANDARD_PROCESS.md#implementation-by-one-plug-in-and-one-host

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  and providing oversight to project administration.
+
+The chair role is assumed to rotate annually, although there are no
+term limits, so the TSC may reelect an existing chair.
+
+The chair will serve as the OpenFX ASWF (Academy Software Foundation)
+TAC (Technical Advisory Council) representative for the term, or may
+delegate this responsibility as needed.
+
+z### TSC Members
```
Comment (SonyDennisAdams) [inline GOVERNANCE.md:113]: Is the "z" in error? I can't find it listed in Markdown syntax.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+* Gary Oberbrunner (chair) - Dark Star Systems
+* Pierre Jasmin - RE:Vision Effects
+* John-Paul Smith - Boris FX
+* Dennis Adams - Sony
```
Comment (SonyDennisAdams) [inline GOVERNANCE.md:118]: Please change to "Sony Creative Software Inc."

**Review by SonyDennisAdams — APPROVED** [review]

**Pull request merged**